app/src/components/page/Sidebar/utils.tsx

```diff
@@ -0,0 +1,24 @@
+import { Route } from '../../../routes';
+
+/**
+ *
+ * @param routes - The routes to get search the active route from
+ * @param pathname - an array of menu location path
+ */
+export function getActiveKey(pathname: string[], routes: Route[]) {
+  const url = '/' + pathname.join('/');
```

Comment:
Cold take location.pathname and avoid the join

Reply:
done